src/App.js

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import List from './components/List';
+import AddItem from './components/AddItem';
+import Nav from './components/Nav';
```

Comment:
Awesome job with changing the `Components` with lowercase `components` here. I still see that the actual folder(directory) name is still `Components` within the app. Everything should work perfectly once you fix it.

Reply:
![Screen Shot 2021-01-09 at 2 19 45 PM](https://user-images.githubusercontent.com/45564632/104089178-ada84900-523a-11eb-880a-c07d4968b17a.png)